packages/cli/src/lib/config/command-object.ts

```diff
@@ -0,0 +1,12 @@
+import { CommandModule } from 'yargs';
+
+export function yargsConfigCommandObject() {
+  return {
+    command: 'config',
+    describe: 'Print config',
+    handler: args => {
+      console.log('Config: ');
+      console.log(args);
```

Comment:
I think it would be useful if this command only emitted valid JSON to the stdout, making it much easier to parse with other tools.

Towards this end, I suggest the following changes:
- drop the `Config: ` line (messes up the JSON formatting and doesn't really need to be reiterated, the user knows what command they ran)
- `console.log`ing an object won't expand nested properties, will include include color codes, etc. - to ensure it's always valid JSON, serialize it with `JSON.stringify`

![image](https://github.com/flowup/quality-metrics-cli/assets/34691111/6a01cad5-c038-4e41-8f74-667eb5196c31)


```suggestion
      console.log(JSON.stringify(args, null, 2));
```